checkout/src/test/java/com/checkout/tokenization/TokenNetworkApiClientTest.kt

```diff
@@ -166,7 +167,7 @@ internal class TokenNetworkApiClientTest {
         @RequiresApi(Build.VERSION_CODES.O)
         @Test
         @DisplayName("Get CardToken Details API does not respond and throw network error")
-        fun apiDoesNotRespond() = runTest {
+        fun apiDoesNotRespond() = runTest(UnconfinedTestDispatcher()) {
```

Comment:
Same as the above comment around using logger 1.0.1 on RIskSdk
If that job is done those `UnconfinedTestDispatcher ` needs to be removed and the tests still work successfully